First pass at documenting integrations (#367)
EventStore/documentation: docs/cloud/integrations/README.md

# Event Store Cloud Integrations

> The Event Store Cloud offers integrations between internal sources such as cluster health and issue detection or notifications events and sinks which include external services such as Slack and OpsGenie.

Reviewer: ```suggestion
Event Store Cloud offers integrations between internal sources such as cluster health, [issue](./issues.md) detection,  [notifications](./notifications.md) events and sinks which include external services such as Slack and OpsGenie.
```

we mostly refer to Event Store Cloud, not The Event Store Cloud

> ## Integration Sources

Reviewer 2: ```suggestion
## Integration sources
```
In other places in docs, we're using capital letters only for the first word.

> "Sources" are driven by events or other mechanism inside the Event Store Cloud.

Reviewer: ```sugggestion
"Sources" are driven by events or other mechanism inside Event Store Cloud.
```

Reviewer 2: **COMMENT:** Events here may be misleading with the business events stored inside EventStoreDB. However, I don't have a better idea so far on how to replace it.

Currently supported sources include:

* [Issues](./issues.md) - issues represent a potentially problematic condition detected inside a cluster or other Event Store Cloud resource. Issues consist of multiple "open" events and a single "closed" event and have different levels of severity.

* [Notifications](./notifications.md) - notifications are note worthy events detected within Event Store Cloud resources or the backend. For example notifications may be emitted when a cluster fails to provision.

> ## Integration Sinks

Reviewer 2: ```suggestion
## Integration sinks
```
